src/index.js

```diff
@@ -1,17 +1,83 @@
-import React from 'react';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+
+import ContactList from './components/contact_list';
+import NewContact from './components/new';
+import ContactInfo from './components/contact_info';
+import axios from "axios";
+
+
+class App extends Component {
```

Comment:
Interesting to do a class component here when you use functional components everywhere else